TextileNode.ts

```diff
@@ -5,6 +5,7 @@ import {
   DeviceEventEmitter,
 } from 'react-native'
 import * as TextileTypes from './App/Models/TextileTypes'
+import { CafeTokens } from './App/Redux/AuthRedux'
```

Comment:
Better to put `CafeTokens` in `TextileTypes.ts`... Trying to consolidate all the types returned by the API in one place so we can eventually provide those definitions externally (from a release of the TextileNode native module package).

Reply:
done